minikube-gradle-plugin/README.md

```diff
@@ -10,16 +10,23 @@ It exposes the following tasks
 - `minikubeStart`
 - `minikubeStop`
 - `minikubeDelete`
+- `minikubeDockerBuild` - runs `docker build` in the minikube Docker environment
 
-It exposes the `minikube` configuration extension.
+It exposes the `minikube` and `docker` configuration extension.
```

Comment:
So it might be a little risky to expose a `docker` extension. It might clash with other docker plugins with an error like "Cannot create 'docker' extension, because it already exists"

Reply:
I you want, you can create the `docker` extension under the `minikube` extension instead of on the project.

Reply:
or just add `docker` it under the minikube extension as a configuration parameter. <-- this might be what I would do.

Reply:
Should we just have the docker client configuration be in minikube extension as well then?

Reply:
yeah, so it looks like ?
```
minikube {
  minikube 'some/path'
  docker = 'some/path'
}

Reply:
Yea, I'll change it to be that way